src/components/App/App.jsx

```diff
@@ -1,53 +1,63 @@
-import React, { useEffect, useState } from 'react'
+import React, { useEffect, useState } from "react";
+import { Route, Switch } from "react-router-dom";
+import PrivateRoute from "../../hoc/PrivateRoute";
 
-import { Route, Switch } from 'react-router-dom'
-import PrivateRoute from '../../hoc/PrivateRoute'
+import { PageView, initGA } from "../Tracking/Tracking";
+import Loading from "../Loading";
+import NavBar from "../NavBar";
+import Footer from "../Footer";
+import Profile from "../Profile";
+import Landing from "../Landing/index.js";
+import DocsPage from "../Docs/index";
+import GridPage from "../GridPage";
+import { useAuth0 } from "../../contexts";
 
-import { PageView, initGA } from '../Tracking/Tracking'
-import Loading from '../Loading'
-import NavBar from '../NavBar'
-import Footer from '../Footer'
-import Home from '../Home'
-import Profile from '../Profile'
-import Landing from '../Landing/index.js'
-import DocsPage from '../Docs/index'
-import GridPage from '../GridPage'
-import { useAuth0 } from '../../contexts'
-
-import './App.scss'
+import "./App.scss";
 
 // fontawesome
-import initFontAwesome from '../../utils/initFontAwesome'
-initFontAwesome()
+import initFontAwesome from "../../utils/initFontAwesome";
+initFontAwesome();
 
 const App = () => {
-  const { loading } = useAuth0()
-  const [apiKey, setApiKey] = useState()
+  const { loading } = useAuth0();
+  const [apiKey, setApiKey] = useState();
+  // const [list, setList] = useState(null)
 
   useEffect(() => {
     /*=== function that initializes Google Analytics ===*/
-    initGA(process.env.REACT_APP_GOOGLE_TRACKING_ID)
-    PageView()
-  })
+    initGA(process.env.REACT_APP_GOOGLE_TRACKING_ID);
+    PageView();
+  });
+
+  // useEffect(()=> {
+  //   axios.get('http://localhost:8888/sauti/client/superlist')
+  //   .then(res => {
+  //     // console.log("in the use effect", res.data)
+  //     setList(res.data)
+  //   })
+  //   .catch(err => {
+  //     console.log(err.message)
+  //   })
+
+  // }, [])
 
   if (loading) {
-    return <Loading />
+    return <Loading />;
   }
 
   return (
-    <div>
-      {' '}
+    <div className="appContainer">
```

Comment:
Not sure what this container is/ what it does.

Reply:
I added that so I could make the footer stay at the bottom of every page.